rbac/processor/role/role_validator.py

```diff
@@ -195,3 +195,61 @@ def _role_already_exists(state_return, role_id):
     return message_accessor.is_in_role_attributes_container(
         container=role_attr_container, identifier=role_id
     )
+
+def get_state_entries(header, confirm, txn_signer_rel_address, state):
```

Comment:
Why move this from state_accessor? It looks like the main goal of the function is to retrieve state entries from the given confirmation.